config/locales/en.yml

```diff
@@ -38,7 +38,7 @@ en:
     pages:
       index:
         landing_page_edit:
-          sections: Edit landing page header section, accordions and timeline
+          sections: Edit landing page header section
```

Comment:
needs to be something like 'Edit landing page header section and accordions'

Reply:
Thanks @Rosa-Fox - fixed up